lib/matplotlib/tests/test_style.py

```diff
@@ -135,6 +152,23 @@ def test_context_with_badparam():
         assert mpl.rcParams[PARAM] == other_value
 
 
+def test_parent_stylesheets():
+    style_dir = os.path.join(os.path.dirname(__file__), 'styles')
+    with temp_style_dir(style_dir):
+        with style.context('main_test'):
+            assert mpl.rcParams['lines.linewidth'] == 5.0
+            assert mpl.rcParams['lines.marker'] == 'o'
+            assert mpl.rcParams['lines.linestyle'] == '--'
+            assert mpl.rcParams['lines.markersize'] == 12
+            assert mpl.rcParams['axes.facecolor'] == 'green'
+            assert mpl.rcParams['axes.linewidth'] == 5.0
+            assert mpl.rcParams['axes.grid'] == True
+            assert mpl.rcParams['axes.labelweight'] == 'bold'
+            assert mpl.rcParams['axes.labelcolor'] == 'yellow'
+            assert mpl.rcParams['axes.color_cycle'] == [u'r', u'g', u'b',
+                                                            u'c', u'y']
```

Comment:
> Ok. Do you think that testing only one parent would be enough?

Unless you have some specific issue in mind, I think the base case (a stylesheet with no parent) and the inductive step (http://en.wikipedia.org/wiki/Mathematical_induction). There could be issues beyond this, but personally, I think its best to deal with them only if there's a specific issue that could present issues.